### PR TITLE
Menu bar Tab

### PR DESCRIPTION
**Pull request by Calvin906**
This is what I have so far for the menu bar tab. It should be good other than the list icon needs to go to the left side.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -10,13 +14,13 @@
         android:theme="@style/AppTheme">
         <activity
             android:name=".HomeScreenActivity"
-            android:theme="@style/Theme.AppCompat.Light.NoActionBar"
-            android:label="@string/app_name"/>
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN"/>
+            android:label="@string/app_name"
+            android:theme="@style/Theme.AppCompat.Light.NoActionBar"/>
```
Comment (JPrez38) [inline app/src/main/AndroidManifest.xml]: home screen activity probably has an action bar no?

File: app/src/main/java/preston/com/recipeapp/RecipeActivity.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+
+/**
+ * Created by Alex Preston on 12/31/15.
+ */
+public class RecipeActivity extends AppCompatActivity{
```
Comment (JPrez38) [inline app/src/main/java/preston/com/recipeapp/RecipeActivity.java]: This class should be a fragment right? Or is that what you are doing now?

File: app/src/main/java/preston/com/recipeapp/SignUpActivity.java [hunk 3]
```diff
@@ -73,12 +81,12 @@ public void beforeTextChanged(CharSequence s, int start, int count, int after) {
...
             @Override
             public void onTextChanged(CharSequence s, int start, int before, int count) {
-            //nothing
+                //nothing
             }
 
             @Override
             public void afterTextChanged(Editable s) {
-            checkSamePassword();
+                checkSamePassword();
```
Comment (JPrez38) [inline app/src/main/java/preston/com/recipeapp/SignUpActivity.java]: huh? i see parameters but you don't pass anything

File: app/src/main/java/preston/com/recipeapp/SignUpActivity.java [hunk 4]
```diff
@@ -35,12 +43,12 @@ public void beforeTextChanged(CharSequence s, int start, int count, int after) {
...
             public void onTextChanged(CharSequence s, int start, int before, int count){
                 //Nothing
-                checkSameEmail();
+
             }
 
             @Override
             public void afterTextChanged(Editable s) {
-
+                checkSameEmail();
```
Comment (JPrez38) [inline app/src/main/java/preston/com/recipeapp/SignUpActivity.java]: same as below

File: app/src/main/res/drawable/rounded_button.xml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
-    <solid android:color="@color/button_color"></solid>
+    <solid android:color="@color/app_extra_color"></solid>
```
Comment (JPrez38) [inline app/src/main/res/drawable/rounded_button.xml]: could prolly use a better name

File: app/src/main/res/menu/menu_bar.xml [hunk 6]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (JPrez38) [inline app/src/main/res/menu/menu_bar.xml]: nice!

File: app/src/main/res/values/colors.xml [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="app_background">#EEEEEE</color>
-    <color name="button_color">#009688</color>
+    <color name="app_extra_color">#009688</color>
```
Comment (JPrez38) [inline app/src/main/res/values/colors.xml]: better name. what is the color?

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -13,6 +13,10 @@
     <string name="signup_confirm_password">Confirm Password </string>
     <string name="signup_pass_dont_match">Passwords don\'t match</string>
     <string name="signup_emails_dont_match">Emails don\'t match</string>
+    <string name="action_list">List</string>
+    <string name="action_vertical">Vertical</string>
```
Comment (JPrez38) [inline app/src/main/res/values/strings.xml]: Whats list and vertical? prolly better names. I assume you mean drawer icon and settings. I also don't get why you need the strings. maybe I'm missing something

File: build.gradle [hunk 9]
```diff
@@ -6,6 +6,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:1.5.0'
+        compile: "com.android.support:appcompat-v7:23.1.0"
```
Comment (JPrez38) [inline build.gradle]: go appcompat!

**Comment by JPrez38** [comment]
make minor changes and you're good to merge!

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Calvin906) [inline app/src/main/AndroidManifest.xml]: So is this done?

File: app/src/main/java/preston/com/recipeapp/RecipeActivity.java (hunk 2 above)
Comment (Calvin906) [inline app/src/main/java/preston/com/recipeapp/RecipeActivity.java]: Yea RecipeActivity got wiped it is now a fragment

File: app/src/main/res/drawable/rounded_button.xml (hunk 5 above)
Comment (Calvin906) [inline app/src/main/res/drawable/rounded_button.xml]: Nah its creative

File: app/src/main/res/values/strings.xml (hunk 8 above)
Comment (Calvin906) [inline app/src/main/res/values/strings.xml]: You said use the RES for strings, and the tutorial I was going off said to use strings

File: build.gradle (hunk 9 above)
Comment (Calvin906) [inline build.gradle]: Dude some of your comments are flabbergasting

**Pull request merged**